lib/roby/cli/log.rb

```diff
@@ -1,23 +1,22 @@
 require 'roby'
 require 'thor'
 
-Roby.app.guess_app_dir
-
 module Roby
     module CLI
         class Log < Thor
             no_commands do
                 def handle_file_argument(file = nil)
-
+                    app = Roby::Application.new
+                    app.guess_app_dir
                     if file
                         return file if File.file?(file)
```

Comment:
You want to initialize an Roby application and guess its dir only if the given file does not exist.  Otherwise avoid to initialize the application

Reply:
Done. Thanks.